### PR TITLE
[visualise] Update visualise to use tkinter instead of pyqt5

### PR DESCRIPTION
**Pull request by jstockwin**
**Description**

Switch from using pyqt5 to using tkinter for the visualise tool.

We're constantly having issues with pyqt5, and it's just generally quite painful to get set up.

Tkinter appears to work with less problems, so we're switching to that. It also allows us to run proper tests for the visualise tool. There is a proof of concept of that in this PR, but more tests should be written. This is tracked in https://github.com/jstockwin/py-pdf-parser/issues/219.

**Linked issues**

None.

**Testing**

Tested locally, and CI tests have been added for the visualise tool.

**Checklist**

- [x] I have provided a good description of the change above
- [x] I have added any necessary tests
- [x] I have added all necessary type hints
- [x] I have checked my linting (`docker-compose run --rm lint`)
- [x] I have added/updated all necessary documentation
- [x] I have updated `CHANGELOG.md`, following the format from
      [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).

### CODE REVIEW

File: py_pdf_parser/visualise/main.py [hunk 1]
```diff
@@ -74,35 +109,51 @@ def __init__(
...
-        page = document.get_page(current_page)
-        self.__fig, self.__ax = self.__initialise_plot(
-            width=page.width, height=page.height
+        self.root = root
+        if width is None:
+            width = self.root.winfo_screenwidth()
+        if height is None:
+            height = self.root.winfo_screenheight()
+        self.root.geometry(f"{width}x{height}")
+        title = f"py-pdf-parser"
```
Comment (paulopaixaoamaral) [inline py_pdf_parser/visualise/main.py:118]: This doesn't need to be an f string.
```suggestion
        title = "py-pdf-parser"
```

File: py_pdf_parser/visualise/main.py [hunk 2]
```diff
@@ -290,6 +317,9 @@ def visualise(
             Defaults to all of the elements in the document.
         show_info (bool): Shows an additional window allowing you to click on
             PDFElements and see details about them. Default: False.
+        width: (int, optional): The initial width of the visualisation window.
```
Comment (paulopaixaoamaral) [inline py_pdf_parser/visualise/main.py:320]: Add comment that default is full width.

File: py_pdf_parser/visualise/main.py [hunk 3]
```diff
@@ -290,6 +317,9 @@ def visualise(
...
+        height: (int, optional): The initial height of the visualisation window.
```
Comment (paulopaixaoamaral) [inline py_pdf_parser/visualise/main.py:321]: Add comment that default is full height.

**Review by paulopaixaoamaral — CHANGES_REQUESTED** [review]
Have a few comments but nothing important. This should be a big improvement to the visualisation tool :+1:

**Comment by jstockwin** [comment]
@pauloAmaral I pushed a new commit

**Review by paulopaixaoamaral — APPROVED** [review]

**Pull request merged**